### PR TITLE
DellInc.Command.Monitor version 10.8.0.284

### PR DESCRIPTION
**Pull request by svenriebedell**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/87528)

### CODE REVIEW

File: manifests/d/DellInc/Command/Monitor/10.8.0.284/DellInc.Command.Monitor.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+PackageName: Dell Command | Monitor
+PackageUrl: https://www.dell.com/support/kbdoc/en-us/000177080/dell-command-monitor
+License: Proprietary
+LicenseUrl: https://www.dell.com/learn/us/en/uscorp1/terms-conditions/art-software-license-agreements
+Copyright: Copyright (C) Dell Inc. 2022. All rights reserved.
+CopyrightUrl: https://www.dell.com/learn/us/en/uscorp1/terms-conditions/trademarks-us
+ShortDescription: The Dell Command | Monitor software application enables IT administrators to easily manage fleet inventory, monitor system health, modify BIOS settings, and remotely collect information for deployed Dell client systems.
+Description: The Dell Command | Monitor software application enables IT administrators to easily manage fleet inventory, monitor system health, modify BIOS settings, and remotely collect information for deployed Dell client systems. Active system health state monitoring can help reduce the total cost of system ownership and is part of a holistic approach to managing all networked devices.
+Tags:
+- Dell Command
```
Comment (Trenly) [inline manifests/d/DellInc/Command/Monitor/10.8.0.284/DellInc.Command.Monitor.locale.en-US.yaml:21]:
```suggestion
- dell-command
```

File: manifests/d/DellInc/Command/Monitor/10.8.0.284/DellInc.Command.Monitor.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: DellInc.Command.Monitor
```
Comment (Trenly) [inline manifests/d/DellInc/Command/Monitor/10.8.0.284/DellInc.Command.Monitor.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: Dell.CommandMonitor
```

File: manifests/d/DellInc/Command/Monitor/10.8.0.284/DellInc.Command.Monitor.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: DellInc.Command.Monitor
```
Comment (Trenly) [inline manifests/d/DellInc/Command/Monitor/10.8.0.284/DellInc.Command.Monitor.yaml:4]:
```suggestion
PackageIdentifier: Dell.CommandMonitor
```

File: manifests/d/DellInc/Command/Monitor/10.8.0.284/DellInc.Command.Monitor.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: DellInc.Command.Monitor
```
Comment (Trenly) [inline manifests/d/DellInc/Command/Monitor/10.8.0.284/DellInc.Command.Monitor.installer.yaml:4]:
```suggestion
PackageIdentifier: Dell.CommandMonitor
```

File: manifests/d/DellInc/Command/Monitor/10.8.0.284/DellInc.Command.Monitor.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: DellInc.Command.Monitor
+PackageVersion: 10.8.0.284
+Installers:
+- Architecture: x64
+  InstallerType: exe
+  InstallerUrl: https://dl.dell.com/FOLDER08796448M/1/Dell-Command-Monitor_KJ0VF_WIN_10.8.0.284_A00.EXE
+  InstallerSha256: B9201ED7A56C3FE70B6B97D2EF33F36F0E5A9EC8BFD5737AFB2431DC5446ABF3
+  InstallerSwitches:
```
Comment (Trenly) [inline manifests/d/DellInc/Command/Monitor/10.8.0.284/DellInc.Command.Monitor.installer.yaml:11]: Please also include SilentWithProgress switch

**Review by Trenly — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request closed without merging**